src/ServiceControl/Notifications/Api/NotificationsController.cs

```diff
@@ -0,0 +1,135 @@
+﻿namespace ServiceControl.Notifications.Api
+{
+    using System;
+    using System.Net;
+    using System.Net.Http;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Web.Http;
+    using System.Web.Http.Results;
+    using Email;
+    using Infrastructure.SignalR;
+    using Newtonsoft.Json;
+    using Raven.Client;
+    using ServiceBus.Management.Infrastructure.Settings;
+
+    public class NotificationsController : ApiController
+    {
+        public NotificationsController(IDocumentStore store, Settings settings)
+        {
+            this.store = store;
+            instanceName = settings.ServiceName;
+        }
+
+        [Route("notifications/email")]
+        [HttpGet]
+        public async Task<JsonResult<EmailNotifications>> GetEmailNotificationsSettings(HttpRequestMessage request)
+        {
+            using (var session = store.OpenAsyncSession())
+            {
+                var settings = await LoadSettings(session).ConfigureAwait(false);
+
+                return new JsonResult<EmailNotifications>(
+                    settings.Email,
+                    new JsonSerializerSettings
+                    {
+                        NullValueHandling = NullValueHandling.Include,
+                        ContractResolver = new UnderscoreMappingResolver()
+                    },
```

Comment:
Don't we have some default json settings that we can reuse? Or make it at least static on the controller?

Reply:
We have a media type registration in `Startup.cs`:

```C#
var jsonMediaTypeFormatter = config.Formatters.JsonFormatter;
                jsonMediaTypeFormatter.SerializerSettings = JsonNetSerializerSettings.CreateDefault();
                jsonMediaTypeFormatter.SupportedMediaTypes.Add(new MediaTypeHeaderValue("application/vnd.particular.1+json"));
```
@tmasternak isn't that enough?

Reply:
No, it's now enough as the default serializer does not align with the ServicePulse conventions we use for naming variables.